src/main/java/com/google/graphgeckos/dashboard/fetchers/CommitData.java

```diff
@@ -0,0 +1,87 @@
+package com.google.graphgeckos.dashboard.fetchers;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Map;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CommitData {
+
+  // name of the branch
+  private String branch;
+
+  /**
+   * Extracts the name of the branch from the reference.
+   * @param ref Git reference
+   */
+  @JsonProperty("ref")
+  private void extractBranch(String ref) {
+    String[] refComponents = ref.split("/");
+    branch = refComponents[refComponents.length - 1];
+  }
+
+  // Git commit hash
+  private String commitHash;
+
+  // time of the Git commit push
```

Comment:
nit: Capitalise time
```suggestion
  // Time of the Git commit push
```